### ISSUE TITLE
Closing manually Facebook auth window gives error

### ISSUE THREAD

**Issue by MaxInMoon** [deepakaggarwal7/react-social-login#13]
```
all.js:109 

Uncaught TypeError: Cannot read property 'data' of undefined
    at SocialLogin.handleSocialLoginInvokeSuccess (main.js:13831)
    at main.js:13899
    at v.__wrapper (all.js:109)
    at ma (all.js:119)
    at Object.w (all.js:117)
    at Object.<anonymous> 

```

**Comment by nicolas-goudry** [comment]
I think it’s because of [this line](https://github.com/deepakaggarwal7/react-social-login/blob/master/src/index.js#L143) from `index.js`.

Can you confirm?

**Comment by nicolas-goudry** [comment]
See https://github.com/deepakaggarwal7/react-social-login/pull/15

**Comment by deepakaggarwal7** [comment]
Fixed with #15 merge
